source/_integrations/zwave_js.markdown

```diff
@@ -55,6 +55,8 @@ To run a Z-Wave network, you need the following elements:
 * A running Z-Wave JS server.
 * An installed Z-Wave integration in Home Assistant.
 
+If you are using Home Assistant Container or Home Assistant Core, refer to the [advanced installation instructions](#advanced-installation-instructions). The next section below does not apply to your setup.
```

Comment:
Because my original phrasing of the section seemed to cause confusion, I wanted to be sure this is very clear. But on second thought, I think you are right. This is probably redundant information here.